### PR TITLE
Test for dimensionality of input in extrapolation routine

### PR DESCRIPTION
**Pull request by katelbach**
in this branch we added a test to check input whether wrong dimensions of precipitation and velocity fields raise errors

### CODE REVIEW

File: pysteps/tests/test_extrapolation_semilagrangian.py [hunk 1]
```diff
@@ -19,3 +19,23 @@ def test_semilagrangian():
...
+
+    invalid_inputs = [
+        (p_1d, v_3d),
+        (p_2d, v_2d),
+        (p_3d, v_2d),
+        (p_3d, v_3d),
+    ]
+    for precip, velocity in invalid_inputs:
+        with pytest.raises(ValueError):
+            extrapolate(precip, velocity, num_timesteps)
```
Comment (chiara-arpae) [inline pysteps/tests/test_extrapolation_semilagrangian.py:44]: please add empty line at the end

**Review by chiara-arpae — COMMENTED** [review]
great job

File: pysteps/tests/test_extrapolation_semilagrangian.py [hunk 2]
```diff
@@ -19,3 +19,23 @@ def test_semilagrangian():
...
+    num_timesteps = 1
+
+    invalid_inputs = [
+        (p_1d, v_3d),
+        (p_2d, v_2d),
+        (p_3d, v_2d),
+        (p_3d, v_3d),
+    ]
+    for precip, velocity in invalid_inputs:
+        with pytest.raises(ValueError):
```
Comment (aperezhortal) [inline pysteps/tests/test_extrapolation_semilagrangian.py:43]: The `pytest` module needs to be imported at the top of the file.

**Comment by dnerini** [comment]
@katelbach, we'll be able to merge your pull request once you address the two minor comments left above. Please let me know if something is not clear.

**Pull request merged**